src/main/resources/io/jenkins/plugins/designlibrary/Localization/index.jelly

```diff
@@ -0,0 +1,68 @@
+<!--
+The MIT License
+
+Copyright (c) 2025, Markus Winter
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in
+all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+THE SOFTWARE.
+-->
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:f="/lib/form" xmlns:s="/lib/samples">
+  <s:layout>
+    <s:section title="${%titlejava}" description="${%howtojava}">
+      <s:group>
+        <s:preview fullWidth="true">
+          <f:entry title="${it.message}">
+            ${it.greet}
+          </f:entry>
+        </s:preview>
+        <s:code-panes>
+          <s:code-pane title="Localize.java">
+            <s:code file="Localize.java"/>
+          </s:code-pane>
+          <s:code-pane title="Messages.properties">
+            <s:code file="Messages.properties"/>
+          </s:code-pane>
+          <s:code-pane title="Messages_de.properties">
+            <s:code file="Messages_de.properties"/>
+          </s:code-pane>
+        </s:code-panes>
+      </s:group>
+    </s:section>
+    <s:section title="${%titlejelly}" description="${%howtojelly}">
+      <s:group>
+        <s:preview fullWidth="true">
+          <f:entry title="${%How to localize texts in Jelly}">
+            ${%hello('World')}
```

Comment:
this is confusing to me that the rendered example doesn't match the example text. (you may need more markup to break this onto separate lines)
```suggestion
            ${%longtext}
            ${%hello('World')}
```